composer.json

```
@@ -81,16 +81,19 @@
     },
 
     "require-dev": {
-        "phing/phing": "^2.16",
-        "phpunit/phpunit": "~4.2",
-
         "behat/behat": "^3.4.3",
         "behat/common-contexts": "^1.2.0",
-        "behat/mink-extension": "^2.3.0",
-        "behat/mink-goutte-driver": "^1.2.1",
         "behat/mink": "^1.7.1",
         "behat/mink-browserkit-driver": "^1.3.2",
-        "behat/mink-selenium2-driver": "^1.3.1"
+        "behat/mink-extension": "^2.3.0",
+        "behat/mink-goutte-driver": "^1.2.1",
+        "behat/mink-selenium2-driver": "^1.3.1",
+        "phing/phing": "^2.16",
+        "phpunit/phpunit": "~4.2"
```

Just for consistency:
```suggestion
        "phpunit/phpunit": "^4.2"
```

---

I will update it in next PR wldedicated for test

---

Also this change is from 
```json
"config": {
        "sort-packages": true
    },	    
```